tools/occlum

```diff
@@ -95,6 +95,20 @@ check_has_built() {
     fi
 }
 
+check_has_run() {
+    if pgrep --full "$instance_dir/build/bin/occlum-run" > /dev/null ; then
+        echo "Error: the process of current Occlum instance is running. Need to wait for the process to finish or kill it first."
+        exit 1
+    fi
+}
+
+check_aesm_service() {
+    if ! pgrep "aesm_service" > /dev/null ; then
```

Comment:
Can we wait a short time (for example 5 seconds) and retry 3 times?

Reply:
It is OK, we should wait a short time in case the asem service has not started yet.